build.sbt

```diff
@@ -1,28 +1,28 @@
-name := "akka-request"
+name := "pekko-request"
 organization := "io.dronekit"
```

Comment:
is it a convenient time to update this too?

Reply:
Yeah, this is going to be semver-major; let's do it.